docs/DocumentationStyleGuide/accessibility.rst

```diff
@@ -0,0 +1,94 @@
+Accessibility
+=============
+
+The moja global community strives towards accessibility in our software
+products and even in the documentation. Through accessibility, we strive
+for successful access to information and resources among people who
+might find it challenging to access them otherwise. We aim to bring
+contributors with limited ability to the forefront of our community to
+have them contribute and access our documentation easily.
+
+The general guidelines for creating accessible documentation pieces are:
+
+-  Maintain consistency across the documentation. Both in style and
+   visuals by employing a fixed hierarchy.
+-  Avoid language or word usage that reeks of bias or a disregard for a
```

Comment:
`reeks` is a strong word for a statement about avoiding biased language :wink: Maybe 'has bias' or 'is potentially biased'?